src/Arcus.Security.Tests.Unit/Core/CachedSecretProviderTests.cs

```diff
@@ -56,7 +55,7 @@ public void CachedSecretProvider_CreateWithoutCache_ShouldSucceed()
             var cacheConfiguration = new CacheConfiguration(TimeSpan.MaxValue);
 
             // Act & Assert
-            new CachedSecretProvider(testSecretProvider, cacheConfiguration);
+            var _ = new CachedSecretProvider(testSecretProvider, cacheConfiguration);
```

Comment:
Just curious, what is the point of this?

Reply:
Some practice I picked up; that you should always provide a value for the ctor to make it more readable.
I wonder though what the point was that was tested here...

Reply:
I don't think we need to capture the response here in a variable.

Reply:
It doesn't matter actually, I just don't like `_` names but let's leave it as is.